### DISCUSSION TITLE
Shadow

### DISCUSSION THREAD

**Discussion by undercoverfemboy** [chakra-ui/chakra-ui#3167] [General] [answered]
Is there any way to change color of boxShadow?

**Comment by chr-ge** [comment, marked as answer]
You can add custom shadows to your chakra theme:

```jsx
const theme = extendTheme({
  shadows: {
    purple: '0 0 0 3px rgba(159, 122, 234, 0.6)'
  }
})
```
You can take a look at the [default shadows here](https://github.com/chakra-ui/chakra-ui/blob/develop/packages/theme/src/foundations/shadows.ts).

**Reply by undercoverfemboy** [reply]
Thanks!

**Reply by mikayelgr** [reply]
The link was not working for me. Here's [the updated and working link](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/foundations/shadows.ts).

**Reply by pettinarip** [reply]
Is it possible to use a color from the theme? something like
```ts
const theme = extendTheme({
  shadows: {
    purple: '0 0 0 3px purple'
  }
})
```
I tried that but it doesn't seem to work :/

**Reply by freshlybakedcode** [reply]
You're going to want to do:
`0 0 0 3px var(--chakra-colors-purple)`

**Reply by mikemajara** [reply]
Link still not working for me https://github.com/chakra-ui/chakra-ui-vue-next/blob/master/packages/theme/src/foundations/shadows.ts

~~@freshlybakedcode did you try that code? Not working either.~~
UPDATE: it doesn't work b/c you need to define purple first (which I didn't have defined)
